src/couch_replicator_scheduler.erl

```diff
@@ -298,7 +312,11 @@ backoff_micros(CrashCount) ->
 
 -spec add_job_int(#job{}) -> boolean().
 add_job_int(#job{} = Job) ->
-    ets:insert_new(?MODULE, Job).
+    InsertRes = ets:insert_new(?MODULE, Job),
+    TotalJobs = ets:info(?MODULE, size),
+    couch_stats:update_gauge([couch_replicator, jobs, total], TotalJobs),
```

Comment:
if insert_new returned false then nothing changed, is it useful to update metrics in that case?